.travis.yml

```diff
@@ -88,33 +43,7 @@ jobs:
     <<: *install-and-test-blurb
   - python: "nightly"  # currently, it's 3.8-dev
     <<: *install-and-test-blurb
-
-  - python: "3.6"
-    <<: *install-and-test-cherry-picker
-  - <<: *base-3_7
-    <<: *install-and-test-cherry-picker
-  - python: "nightly"  # currently, it's 3.8-dev
-    <<: *install-and-test-cherry-picker
-  - os: windows
-    language: sh
-    python: 3.7
-    before_install:
-      - choco install python --version 3.7
-      - python -m pip install --upgrade pip wheel
-      - *install-flit
-    <<: *install-and-test-cherry-picker
-    env:
-      PATH: >-
-        /c/Python37:/c/Python37/Scripts:$PATH
-      TARGET_PKG: cherry_picker
-
-  - <<: *deploy-base
-    <<: *run-if-cherry-picker
-    env:
-      TARGET_PKG: cherry_picker
-
-  - <<: *deploy-base
```

Comment:
@Mariatta I believe that the deploy base shouldn't have been removed from here. Of course, it doesn't affect anything because auto-deployment hasn't been enabled for blurb. But if anybody will want to use it, they'd have to recover that base mapping.